### DISCUSSION TITLE
Documentation is down

### DISCUSSION THREAD

**Discussion by martinsinn** [SaltyAom/niku#10] [Q&A] [answered]
Hey folks! Loving the concept. Can ou make the documentation accessible again?

**Comment by SaltyAom** [comment, marked as answer]
Hello, martinsinn!
Sorry for the inconvenience of the API server is down, you can use the [mirror site](https://niku-docs.netlify.app) for now.

TLDR; the normal server should be back in 3-5 days, please use [mirror site](https://niku-docs.netlify.app) for now instead.

Originally, the docs are deployed on Vercel but for some reason, after I wake up this morning, they suspense my account without any future notice and result in so sudden which results in the API is down.
I'm contacting Vercel for further action to see what I can do, for now, please use the [mirror site](https://niku-docs.netlify.app/) which might respond to me in 3-5 business working days, however, I'm migrating most of my projects which is web-based to either Netlify or Google Cloud which should resolve the problem.

Unfortunately, there's a lot to set up for moving every project out, so I'm gradually doing it after my career shift end.
Thanks for using Niku, cheers!

**Comment by SaltyAom** [comment]
Hi, just here for an update.
The [main server](https://niku.saltyaom.com) server is back!
I've contacted Vercel and decided to move out. The server is now running on Netlify instead.
Cheers 🎉
